liquid/static/css/sponsors.css

```diff
@@ -14,5 +19,5 @@
 .company-card img {
   margin: 0 auto;
   display: block;
-  max-height: 100%;
+  max-height: 46px; /* For oversized logos; TODO ask companies for consistent sizes */
```

Comment:
Instead of asking for a certain size we should just rescale. If not in CSS, we can do it on image upload.